HW3-JS-Muscle-Memory/app.js

```diff
@@ -0,0 +1,828 @@
+// Task 1 - Notes for JS Evaluation Predictions
+
+// List 3 expressions that surprised you or gave unexpected results, why?
+2.5 * "fish;"; // I figured it would be undefined or NaN because in my head I'm like you can't do that
+2 + "pizza"; // this one was surpising as well but I guess adding it is different then multiplyin
+
+let str = "General Assembly";
+if (str[0] === str[0].toUpperCase()) {
+  console.log("starts with a capital!");
+}
+if (str[str.length - 1] === str[str.length - 1].toUpperCase()) {
+  console.log("ends with a capital!");
+} // I don't understand this one... it would log ends with a caputal but there are no ending capitals..idk
```

Comment:
When I copy and paste this code, I get `starts with a capital!` something could have been off with the rest of the file that you had which could have resulted in getting that unexpected result.